docs/sources/alerting/set-up/configure-alertmanager/index.md

```diff
@@ -11,22 +15,51 @@ keywords:
   - external Alertmanager
 labels:
   products:
+    - cloud
     - enterprise
     - oss
 title: Add an external Alertmanager
 weight: 200
+refs:
+  notifications:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/notifications/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/notifications/
 ---
 
-# Add an external Alertmanager
+## Grafana Alertmanager
```

Comment:
can we call it Configure Alertmanagers? it is for grafana alertmanagers and external alertmanagers this topic now since you added the other info right?

Reply:
Addressed by https://github.com/grafana/grafana/pull/88567/commits/7cb6bffc17dd2cb3481bb9cf99f9e9f6f32a81f1